client/image_inspect.go

```diff
@@ -24,6 +24,10 @@ func (cli *Client) ImageInspect(ctx context.Context, imageID string, inspectOpts
 		}
 	}
 
+	if opts.apiOptions.Manifests && opts.apiOptions.Platform != nil {
+		return image.InspectResponse{}, fmt.Errorf("manifests and platform options cannot both be set")
+	}
+
```

Comment:
If we _do_ want to keep the error, we should probably remove this check in the client, and leave it to the API server to return an error

Reply:
Good call